src/app/examples/loading-overlay-example/loading-overlay-example.component.html

```diff
@@ -0,0 +1,10 @@
+<kirby-loading-overlay [isLoading]="isLoading" [showBackdrop]="showBackdrop">
```

Comment:
This goes for both {N} and web example.

What are all the alert buttons good for? are they just not more noisy then given value to the example?

Reply:
It is to demonstrate you can't click them when the overlay is covering. :)

Reply:
Well you already have 2 buttons to show the loading overlay. I would delete the alert buttons. In my opinion they are create to much noice in the example 😄

Reply:
I ended up only having one ;)